src/api/genes.ts

```diff
@@ -32,3 +30,30 @@ export const search = async (
     return [];
   }
 };
+
+// search genes by keyword
+export const batchSearch = async (
+  query?: string[],
+  species?: string[]
+): Promise<Gene[]> => {
+  // params
+  const params = new URLSearchParams();
+  if (query?.length) params.set("q", query.map(e => `"${e}"`).join(","));
+  if (species?.length) params.set("species", species.join(","));
+  params.set("fields", "*");
```

Comment:
I think it would be good to add another parameter to batch searches: `scopes=_id,symbol,entrezgene,ensembl,uniprot` (not name.... since it can have ambiguous matches), to allow multiline entry of other identifiers.

Reply:
*edit: I added "alias". The field contains other commonly used aliases for the gene.

Reply:
Should I add this to the regular single search as well?

Reply:
Also I'm putting quotes around spaced queries like this `q="ajuba LIM protein","glucose"`. Otherwise it seems like the backend splits them into separate searches by space like `q=ajuba,LIM,protein,glucose". Is this correc.t

Reply:
I also added Aliases as a column to the gene tables. I think that is useful, or at least as useful as the other columns.

Reply:
The single search already searches the `alias` field by default.

Reply:
> Also I'm putting quotes around spaced queries like this q="ajuba LIM protein","glucose". Otherwise it seems like the backend splits them into separate searches by space like q=ajuba,LIM,protein,glucose. Is this correct

Yes, that's also an issue I was worried about with regards to batch searching the 'name' field... I don't think quotation marks work either. Is there a specific query you were testing?

Reply:
With regards to displaying the `alias` field, I think it takes a lot of space and forces a horizontal scroll. Aliases are unofficial gene symbols, so while they are useful for searching, we prefer to store the correct `symbol` in the genesets.

Reply:
Adding the quotes was just totally a guess. How are users able to search for multiple words as one search?

Reply:
Just found out. Replace spaces with dashes `-`.
```

curl -s -X POST "https://mygene.info/v3/query?q='cyclin-dependent-kinase-12'&scopes=name&species=9606" |jq[
  {
    "query": "cyclin-dependent-kinase-12",
    "_id": "51755",
    "_score": 24.52365,
    "entrezgene": "51755",
    "name": "cyclin dependent kinase 12",
    "symbol": "CDK12",
    "taxid": 9606
  }
]
```